docs/1-getting-started/0-understand-the-basics.md

```diff
@@ -16,7 +16,7 @@ Before you can effectively use Zarf, it is useful to have an understanding of th
 ### AirGap Basics
 
 - [What is AirGap](https://ibm.github.io/kubernetes-networking/vpc/airgap/)
-- AirGap Kubernetes Course - Coming Soon!
+- [AirGap Kubernetes Course](https://training.linuxfoundation.org/training/modern-air-gap-software-delivery-lfs281/) - Created by Defense Unicorns
```

Comment:
I put in a Created by Defense Unicorn Note here cause I thought we may want to people to know that. Not sure though - thoughts @Racer159

Reply:
I think that is fine, but @andrewg-xyz / @TheFutonEng are their any attribution requirements from LF?

Reply:
If they have an issue with it we will PR it back per convo